client/assets/components/login/login.js

```
@@ -30,6 +30,14 @@
 
     vm.submit = submit;
 
+    init();
+
+    function init() {
+      if (AuthService.hasSamlRealm()) {
+        AuthService.getSession().catch(AuthService.authBySaml);
```

(Super minor) Even tho this is shorter, it may be more concise to use a lambda in the callback so it is immediately obvious we are intentionally swallowing the error and calling authBySaml where now it looks like we are intentionally passing the error parameter to authBySaml.